frontend/src/components/Locations/LocationForm.tsx

```diff
@@ -212,7 +226,9 @@ const LocationForm = ({current, createNew, setLocation, PickupDeliveryObject, se
         )}
 
         <div>
-          <button className="bg-amber-500 rounded-full w-full mt-5 p-3 text-3xl text-white font-semibold shadow" onClick={() => submitPressed()}>Continue</button>
+          <button className="bg-amber-500 rounded-full w-full mt-5 p-3 text-3xl text-white font-semibold shadow"
+          disabled= {!isClicked}
```

Comment:
If you type something into the 'other' box then delete it, it will still allow you to submit the form. (It will also accept whitespace) 

I'm not super sure what's going on with the isClicked vs isOtherClicked but I think isClicked might not make sense to use anymore given we're using checkboxes instead of radio buttons. It might be a good idea to rename it or use a different state variable that keeps track of whether the entire form is valid or not.

Reply:
It seems like whatever is being stored in other isn't getting added to the user's selections. I added `console.log(active);` to line 62 and this is what's getting printed out. Let me know if you can reproduce this error. 

<img width="1440" alt="Screen Shot 2023-02-21 at 12 59 26 PM" src="https://user-images.githubusercontent.com/70536670/220458844-8b687f60-6ab4-4acd-9e5e-973ac47fb7d3.png">

<img width="1440" alt="Screen Shot 2023-02-21 at 12 59 35 PM" src="https://user-images.githubusercontent.com/70536670/220458853-c1c4a5ea-3ca1-448a-9c54-d1247d007393.png">